firefox_puppeteer/ui/update_wizard/wizard.py

```diff
@@ -0,0 +1,347 @@
+        """The finished panel.
+
+        :returns: :class:`FinishedPanel` instance.
+        """
+        return self._create_panel_for_id('finished')
+
+    @property
+    def finished_background(self):
+        """The finished background panel.
+
+        :returns: :class:`FinishedBackgroundPanel` instance.
+        """
+        return self._create_panel_for_id('finishedBackground')
+
+    @property
+    def incompatible_check(self):
+        """The incompatible check panel.
+
+        :returns: :class:`IncompatibleCheckPanel` instance.
+        """
+        return self._create_panel_for_id('incompatibleCheck')
+
+    @property
+    def incompatible_list(self):
+        """The incompatible list panel.
+
+        :returns: :class:`IncompatibleListPanel` instance.
+        """
+        return self._create_panel_for_id('incompatibleList')
+
+    @property
+    def installed(self):
+        """The installed panel.
+
+        :returns: :class:`InstalledPanel` instance.
+        """
+        return self._create_panel_for_id('installed')
+
+    @property
+    def license(self):
+        """The license panel.
+
+        :returns: :class:`LicensePanel` instance.
+        """
+        return self._create_panel_for_id('license')
+
+    @property
+    def manual_update(self):
+        """The manual update panel.
+
+        :returns: :class:`ManualUpdatePanel` instance.
+        """
+        return self._create_panel_for_id('manualUpdate')
+
+    @property
+    def no_updates_found(self):
+        """The no updates found panel.
+
+        :returns: :class:`NoUpdatesFoundPanel` instance.
+        """
+        return self._create_panel_for_id('noupdatesfound')
+
+    @property
+    def plugin_updates_found(self):
+        """The plugin updates found panel.
+
+        :returns: :class:`PluginUpdatesFoundPanel` instance.
+        """
+        return self._create_panel_for_id('pluginupdatesfound')
+
+    @property
+    def updates_found_basic(self):
+        """The updates found panel.
+
+        :returns: :class:`UpdatesFoundPanel` instance.
+        """
+        return self._create_panel_for_id('updatesfoundbasic')
+
+    @property
+    def updates_found_billboard(self):
+        """The billboard panel shown if an update has been found.
+
+        :returns: :class:`UpdatesFoundBillboardPanel` instance.
+        """
+        return self._create_panel_for_id('updatesfoundbillboard')
+
+    @property
+    def panels(self):
+        """List of all the available :class:`Panel` instances.
+
+        :returns: List of :class:`Panel` instances.
+        """
+        panels = self.marionette.execute_script("""
+          let wizard = arguments[0];
+          let panels = [];
+
+          for (let index = 0; index < wizard.children.length; index++) {
+            if (wizard.children[index].id) {
+              panels.push(wizard.children[index].id);
+            }
+          }
+
+          return panels;
+        """, script_args=[self.element])
```

Comment:
You might need filters for this method. You may not want all the children, maybe some by id or others via classname. So once we have a good confidence with the deck/wizard code we could make a generic class for it. Would be good to re-use the code given that we already have nearly 3 identical copies around in our code.